lib/cdn-site-hosting/cdn-site-hosting-props.ts

```diff
@@ -1,5 +1,6 @@
 import * as cdk from "@aws-cdk/core";
 import * as s3deploy from "@aws-cdk/aws-s3-deployment";
+import { ResponseSecurityHeadersBehavior } from "@aws-cdk/aws-cloudfront/lib/response-headers-policy";
```

Comment:
nitpick: to avoid referencing the internals of the package, which may change in future versions

```suggestion
import { ResponseSecurityHeadersBehavior } from "@aws-cdk/aws-cloudfront";
```

Reply:
Done!